webapp/components/hooks/useNotifyError.js

```diff
@@ -0,0 +1,3 @@
+import { useNotification } from './useNotification'
+
+export const useNotifyError = () => useNotification({ severity: 'error' })
```

Comment:
maybe this hooks can be a subtype of useNotification and can be added into 

useNotification/
...index.js
...useNotificationError 

if not the components/hooks could grow without any sense

Reply:
uhm, I think I've understood!